CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.8.2 (2021-07-08)
+
+- Updated styles for AG Grid 24 with modern theme. [#121](https://github.com/blackbaud/skyux-ag-grid/pull/121)
```

Comment:
```suggestion
- Updated styles for AG Grid version 24 in modern theme. [#121](https://github.com/blackbaud/skyux-ag-grid/pull/121)
```
According to the changelog, we just added support for version 25 at the beginning of June. So why are updating styles for version 24 only if we also support version 25?

Reply:
Should have been "version 24 and above."